helpers/config.py

```diff
@@ -820,8 +822,10 @@ def __questions_postgres(self):
 
         Settings can be tweaked thanks to pgconfig.org API
         """
-        self.__config["postgres_db"] = CLI.colored_input("Postgres database", CLI.COLOR_SUCCESS,
-                                                         self.__config.get("postgres_db"))
+        self.__config["kc_postgres_db"] = CLI.colored_input("KoBoCat PostgreSQL database", CLI.COLOR_SUCCESS,
+                                                         self.__config.get("kc_postgres_db"))
```

Comment:
@noliveleger notes that existing installs will need this to be set to the value of `postgres_db`, so we could try something like `self.__config.get("postgres_db", self.__config.get("kc_postgres_db"))`